modules/storage-page-memory/src/test/java/org/apache/ignite/internal/storage/pagememory/mv/PersistentPageMemoryMvPartitionStorageConcurrencyTest.java

```diff
@@ -52,7 +52,7 @@ void setUp(
             @InjectConfiguration("mock.checkpoint.checkpointDelayMillis = 0")
             PersistentPageMemoryStorageEngineConfiguration engineConfig,
             @InjectConfiguration("mock.profiles.default = {engine = \"aipersist\"}")
-            StorageConfiguration storageConfiguration
+            StorageConfiguration storageConfig
```

Comment:
It's strange that you've shortened this name. What motivated you?

Reply:
For consistency in fields configuration naming, previously there was only `engineConfig`, then they brought `storageConfiguration`, it confused me a little, I can do it in a separate task.

Reply:
I'm fine with it being here